Source/Randomizer.cpp

```diff
@@ -403,11 +403,15 @@ void Randomizer::RandomizeChallenge() {
 void Randomizer::RandomizeAudioLogs() {
     std::vector<int> randomOrder(audiologs.size(), 0);
     std::iota(randomOrder.begin(), randomOrder.end(), 0);
-    Randomize(randomOrder, SWAP::NONE);
+    RandomizeInPlace(randomOrder, SWAP::NONE);
     ReassignNames(audiologs, randomOrder);
 }
 
-void Randomizer::Randomize(std::vector<int>& panels, int flags) {
+void Randomizer::Randomize(std::vector<int> panels, int flags) {
+    return RandomizeInPlace(panels, flags);
+}
+
+void Randomizer::RandomizeInPlace(std::vector<int>& panels, int flags) {
```

Comment:
I'd like to pull this bugfix into the main branch as well -- I'm happy to cherry-pick this, or you can create a second PR, either works.